create CODE_OF_CONDUCT.md

vlasvlasvlas opened the pull request:
hope we can get a code of conduct for issues! thanks :D

---
Make sure to mark the boxes below before creating PR: [x]

- [ ] Description above provides context of the change
- [ ] Unit tests coverage for changes (not needed for documentation changes)
- [ ] Target Github ISSUE in description if exists
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ ] Relevant documentation is updated including usage instructions.
- [ ] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

vlasvlasvlas committed:
create CODE_OF_CONDUCT.md

hope we can get a code of conduct for issues! thanks :D

vlasvlasvlas committed:
Update CODE_OF_CONDUCT.md

more like this

feluelle reviewed (commented):
Agree with Fokko. I think we should just link to the Apache CoC. WDYT?

vlasvlasvlas committed:
Update CODE_OF_CONDUCT.md

following @turbaszek tips and this chat about adding a code of conduct https://github.com/apache/airflow/pull/9715 :)

potiuk reviewed (approved):
Looks Good but I believe licence header is missing - you can auto-generate it with pre-commits  (See STATIC_CHECKS.rst)

Fokko commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,5 @@
+# Contributor Covenant Code of Conduct
+
+The Apache Airflow project follows the [Apache Software Foundation code of conduct](http://www.apache.org/foundation/policies/conduct.html).
```
Minor, could we make this http**s**?

vlasvlasvlas replied:
sorry for the delay, ill change that

turbaszek commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,5 @@
+
+The Apache Airflow project follows the [Apache Software Foundation code of conduct](http://www.apache.org/foundation/policies/conduct.html).
+
+If you observe behavior that violates those rules please follow the [ASF reporting guidelines](https://www.apache.org/foundation/policies/conduct#reporting-guidelines). 
```
```suggestion
If you observe behavior that violates those rules please follow the [ASF reporting guidelines](https://www.apache.org/foundation/policies/conduct#reporting-guidelines).
```

vlasvlasvlas committed:
Update CODE_OF_CONDUCT.md

Co-authored-by: Tomek Urbaszek <turbaszek@gmail.com>

turbaszek commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,5 @@
+# Contributor Covenant Code of Conduct
```
```suggestion
<!--
 Licensed to the Apache Software Foundation (ASF) under one
 or more contributor license agreements.  See the NOTICE file
 distributed with this work for additional information
 regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing,
 software distributed under the License is distributed on an
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
-->

# Contributor Covenant Code of Conduct
```

turbaszek commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,5 @@
+# Contributor Covenant Code of Conduct
+
+The Apache Airflow project follows the [Apache Software Foundation code of conduct](http://www.apache.org/foundation/policies/conduct.html).
```
```suggestion
The Apache Airflow project follows the [Apache Software Foundation code of conduct](https://www.apache.org/foundation/policies/conduct.html).
```

vlasvlasvlas committed:
Update CODE_OF_CONDUCT.md

Co-authored-by: Tomek Urbaszek <turbaszek@gmail.com>

turbaszek committed:
Update CODE_OF_CONDUCT.md
